cEP-0035.md

```diff
@@ -0,0 +1,52 @@
+# Extending support for newer versions of Python
+
+| Metadata |                                       |
+| -------- | ------------------------------------- |
+| cEP      | 0035                                  |
+| Version  | 1.0                                   |
+| Title    | Extending support for newer version of Python  |
+| Authors  | Anshal Shukla <mailto:shukla.anshal85@gmail.com> |
+| Status   | Proposed                              |
+| Type     | Process                               |
+
+## Abstract
+
+This cEP describes the upgrades and steps required to enable coala to support newer versions of Python. This project has been taken up as a [GSoC](https://summerofcode.withgoogle.com/organizations/6490386047959040/#4634486723051520) project.
```

Comment:
Mention the GSoC year, also this line is too long, please use a decent max line length.